src/components/common/PwaPopUp.jsx

```
@@ -0,0 +1,189 @@
+      }
+    }
+
+    setDeferredPrompt(null)
+  }
+
+  const handleDontShowToday = (e) => {
+    setDontShowToday(e.target.checked)
+    console.log('PWA :하루동안 안보기 1')
+  }
+
+  const handleClose = (e) => {
```

ESLint

[error] 'e' is defined but never used.

(no-unused-vars)